chart/templates/gateway/gcp-backend-policy.yaml

```diff
@@ -0,0 +1,17 @@
+
+# FIXME: Delete when gateway support tiemouts
+# https://gateway-api.sigs.k8s.io/geps/gep-1742/
```

Comment:
Can we create a story for it so that we don't lose track of this one? 😄

Reply:
created 👍🏼